components/popover/package.json

```diff
@@ -17,19 +17,19 @@
   },
   "peerDependencies": {
     "@spectrum-css/dialog": "^6.0.0",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^1.0.4"
```

Comment:
I think we are now good to use version `1.0.7`

Reply:
Fixed in latest commit.